gateway/handle.go

```diff
@@ -271,8 +271,11 @@ func (gw *Handle) getJobDataFromRequest(req *webRequestT) (jobData *jobFromReq,
 
 	fillMessageID := func(event map[string]interface{}) {
 		messageID, _ := event["messageId"].(string)
-		if strings.TrimSpace(messageID) == "" {
+		messageID = misc.SanitizeUnicode(messageID)
```

Comment:
space trimming has been intentionally omitted?

Reply:
It's probably a good idea to have both. I was thinking that given the `SanitizeUnicode` the additional trim can be an overkill, but I can think of cases where just empty spaces can be confusing